great_expectations/rule_based_profiler/domain_builder/categorical_column_domain_builder.py

```diff
@@ -17,16 +29,16 @@
     get_parameter_value_and_validate_return_type,
     get_resolved_metrics_by_key,
 )
-from great_expectations.rule_based_profiler.parameter_container import (
-    ParameterContainer,  # noqa: TCH001
-)
-from great_expectations.validator.computed_metric import MetricValue  # noqa: TCH001
 from great_expectations.validator.metric_configuration import MetricConfiguration
 
 if TYPE_CHECKING:
     from great_expectations.data_context.data_context.abstract_data_context import (
         AbstractDataContext,
     )
+    from great_expectations.rule_based_profiler.parameter_container import (
+        ParameterContainer,  # noqa: TCH001
+    )
+    from great_expectations.validator.computed_metric import MetricValue  # noqa: TCH001
```

Comment:
```suggestion
        ParameterContainer
    )
    from great_expectations.validator.computed_metric import MetricValue
```

Please remove these comments

Reply:
@Kilo59 Committed -- thanks.